config/armada/config.yaml

```diff
@@ -54,6 +54,8 @@ scheduling:
   maximumJobsToSchedule: 5000
   maxQueueReportsToStore: 1000
   MaxJobReportsToStore: 100000
+  resourceScarcity:
```

Comment:
is adding this going to change the behaviour in prod clusters- e.g. for a gpu pool which only had scarcity set for gpus, it'll now start considering cpus?

Reply:
Per-pool constraints trump the global setting.